packages/google_maps_flutter/google_maps_flutter_ios/example/shared/maps_example_dart/lib/example_google_map.dart

```diff
@@ -214,15 +214,13 @@ class ExampleGoogleMap extends StatefulWidget {
     this.onMapCreated,
     this.gestureRecognizers = const <Factory<OneSequenceGestureRecognizer>>{},
     this.compassEnabled = true,
-    this.mapToolbarEnabled = true,
```

Comment:
These changes to the iOS example are opportunistic cleanup; these have always been Android-only, and just got copypasta'd here when I federated the implementations out of the original shared implementation.